ccloud-monitoring/docs/consumer-scenarios/consumer-lag-problem.rst

```diff
@@ -0,0 +1,164 @@
+.. _ccloud-monitoring-consumer-connectivity-problem:
+
+Consumer Lag Problem
+********************
+
+Consumer group lag is a tremendous performance indicator. It tells you the difference between the
+producer's last produced message and the consumer group's last commit. If you are unfamiliar with
+consumer groups or concepts like committing offsets, please refer to this
+`Kafka Consumer documentation <https://docs.confluent.io/platform/current/clients/consumer.html>`__.
+
+A large or quickly growing lag indicates that the consumer is not able to keep up with
+the volume of messages on a topic.
+
+This scenario will look at Confluent Cloud metrics from the Metrics API, `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ metrics, and
+client metrics from the client application’s MBean object ``kafka.consumer:type=consumer-fetch-manager-metrics,client-id=<client_id>``.
+
+Introduce failure scenario
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Stop ``consumer-1`` container, thus removing a consumer from the consumer group and add 2 producers:
```

Comment:
- Presumably this command stops ``consumer-1`` because the scale is dropped from 2 to 1, but maybe it's worth an explanation.  I had to do a double-take on this to understand that.
- Add `2` producers?
- Maybe it's worth a reminder to say, something like, by default, X producer and Y consumers are running.  Adjust that to X' and Y' in order to show...

Reply:
👍